test/modules/http/response/http-response-readable-stream.test.ts

```diff
@@ -29,13 +29,16 @@ afterAll(async () => {
 
 it('supports ReadableStream as a mocked response', async () => {
   const encoder = new TextEncoder()
+  const chunks = ['hello', ' ', 'world']
   interceptor.once('request', ({ controller }) => {
     const stream = new ReadableStream({
-      start(controller) {
-        controller.enqueue(encoder.encode('hello'))
-        controller.enqueue(encoder.encode(' '))
-        controller.enqueue(encoder.encode('world'))
-        controller.close()
+      pull(controller) {
+        const chunk = chunks.shift()
+        if (chunk) {
+          controller.enqueue(encoder.encode(chunk))
+        } else {
+          controller.close()
+        }
```

Comment:
Using `pull` is a more realistic way of creating a streaming response than using `start`